content/authors/zhenjieweng/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+# Display name
+title: 翁振杰
+
+# Username (this should match the folder name)
+authors:
+- zhenjieweng
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: 2020硕
+
+# Organizations/Affiliations
+organizations:
+- name: East China Normal University
+  url: "http://www.ecnu.edu.cn/"
+
+# Short bio (displayed in user profile at end of posts)
+bio: 技术传播文明，践行同步世界
+
+interests:
+- playing chess
+- reading
+
+
+education:
+  courses:
+  - course: MEng in Computer technique
+    institution: East China Normal University
+    year: 2023
+  - course: BEng in Information Management and Information System
```

Comment:
Information Management and Information System is a major of management, while BEng means bachelor of engineering.

Reply:
remember using `git commit --amend` to commit your changes afterwards.